.github/workflows/alert-sync-issues.yml

```
@@ -0,0 +1,16 @@
+name: Alert sync issues
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)